src/components/application_manager/test/resumption/resumption_sql_queries_test.cc

```diff
@@ -234,7 +235,8 @@ class ResumptionSqlQueriesTest : public ::testing::Test {
   SQLQuery& FillSubMenuTable(SQLQuery& query,
                              const int menuID,
                              const string& menuName,
-                             const int position);
+                             const int position,
+                             const int64_t idimage);
```

Comment:
@VVeremjova What does it mean `idimage`? If it is id for image - rename variable to `image_id`